code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -31,6 +31,12 @@
 	var/notification_sound = TRUE // Whether the bracer pings when a message comes or not
 	var/charge = 1500
 	var/charge_max = 1500
+	/// The amount charged per process
+	var/charge_rate = 30
+	/// Cooldown on draining power from APC
+	var/charge_cooldown = 3 MINUTES
+	/// Worldtime of last recharge
+	var/last_charge
```

Comment:
use the cooldown defines for managed cooldowns, either COOLDOWN_ or TIMER_COOLDOWN_